pkg/kedge/common/common.go

```diff
@@ -89,3 +87,11 @@ func PortAllowed(port int, portRule *pb.Adhoc_Port) bool {
 	}
 	return false
 }
+
+func ResolveHost(hostStr string) (string, error) {
+	addrs, err := DefaultALookup(hostStr)
+	if err != nil {
+		return "", err
+	}
+	return addrs[0], nil
```

Comment:
is there ever a case where `err == nil` but `len(addrs) == 0`? not to familiar with the net package

Reply:
good point, looking at other usages in other packages this can happen. will handle that case

Reply:
cc @Bplotka

Reply:
This is very good point - I don't know about net package but this is interface so we need to expect anything!